src/App.jsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react'
+import Explore from './pages/Explore'
```

Comment:
rename the folder pages please, you don't have pages in React since it is a single page application. 
If you don't like to work with `components` then go with `views`